docs/articles/apex/writing.md

```diff
@@ -26,51 +37,58 @@ You can preview, view, or go to definitions of:
   - Variables (local and class variables)
 - Standard objects
   - Fields (standard and custom fields)
-  - Object definitions
+  - Object definition
 - Custom objects
   - Fields
   - Object definitions
 
-(See the [Enable Code Smartness for SObjects](#enable-code-smartness-for-sobjects) section of this topic for information on working with standard and custom objects.)
+When you refresh SObject definitions, the extension uses the default org to generate Apex classes. These classes represent the standard and custom objects that the current user has access to. **Do not** edit these representative Apex classes, which are stored in a hidden directory on your local workstation.
```

Comment:
```suggestion
When you refresh SObject definitions, the extension uses the default org to generate Apex classes. These classes represent the standard and custom objects that the current user has access to. **Do not** edit these representative Apex classes, which are stored in the hidden `.sfdx` directory in your project.
```

Reply:
I'm torn about saying "__do not__" edit these classes if the user knows what they're doing. I'd maybe say it isn't recommended and explain that they're intended to be read-only and if they're modified, it may impact the autocomplete experience and any changes made may be wiped when rerunning the refresh command.